.travis.yml

```diff
@@ -17,6 +17,9 @@ script:
 - yarn upgrade @angular/compiler@~10.0.0
 - yarn run test
 
+- yarn upgrade @angular/compiler@~10.1.0
```

Comment:
Since `10.2.x` is also available, it'd be better to test `^10.1.0` I guess.

```suggestion
- yarn upgrade @angular/compiler@^10.1.0
```

Reply:
I did checked all versions... mush be blind